omc3/optics_measurements/phase.py

```diff
@@ -226,11 +229,16 @@ def _get_square_data_frame(data, index):
     return pd.DataFrame(data=data, index=index, columns=index)
 
 
-def write_special(meas_input, phase_advances, plane_tune, plane):
+def write_special(meas_input: DotDict, phase_advances, plane_tune, plane):
     # TODO REFACTOR AND SIMPLIFY
-    accel = meas_input.accelerator
+    accel: Accelerator = meas_input.accelerator
+
+    important_phase_advances: list = accel.important_phase_advances()
+    if not important_phase_advances:
+        return
```

Comment:
Let's maybe specify in the docstring or so that this function **might** not do anything, in case there are no important phase advances for the given accelerator.